Hush/Assets/Scripts/Game/GameManager.cs

```diff
@@ -21,6 +22,11 @@ private void Awake()
             DontDestroyOnLoad(Instance.gameObject);
 
             Instance.InitializeValues();
+            
+        #if UNITY_EDITOR
+            SetActiveLightSpell(SpellType.FireballSpell);
+            SetActiveHeavySpell(SpellType.FireballSpell);
+        #endif
```

Comment:
going to conflict with master's SceneManager

Reply:
Removed in 61c6a87, was only a hack to avoid setting spells in `Start()` all the time